geocity/apps/permits/models.py

```diff
@@ -940,6 +940,16 @@ def get_works_type_names_list(self):
             )
         )
 
+    def get_works_object_names_list(self):
+
+        return ", ".join(
+            list(
+                self.works_object_types.all()
+                .values_list("works_object__name", flat=True)
```

Comment:
Use new shortname field ?

Reply:
> Use new shortname field ?

No, as specified here https://jira.liip.ch/browse/YC-883 we want the object name (we already have the type name in the email subject)

> How does the title gets cropped if many WOT are selected ?

Like the type names in the email subject: concatenated and separated by commas.